tests/switchless/enc/CMakeLists.txt

```diff
@@ -0,0 +1,15 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+
+add_custom_command(
+  OUTPUT switchless_t.h switchless_t.c switchless_args.h
+  DEPENDS ../switchless.edl
```

Comment:
You'll want `edger8r` in the `DEPENDS` list otherwise when developing the edger8r you run into dependency issues 😅

Reply:
Noted.